src/navigation/SettingsStack.tsx

```diff
@@ -46,6 +47,10 @@ const SettingsStack: FunctionComponent = () => {
         }}
```

Comment:
The header shown detail might be useful for the rest of the views under this stack but we will most likely have to set the options with the hook once the screen loads to update the title and maybe the back action?